src/encoded/static/components/typeutils.js

```diff
@@ -460,3 +461,84 @@ ControllingExperiments.defaultProps = {
     total: 0,
     url: '',
 };
+
+/**
+ * Display a count of biosample characterizations in the footer, with a link to the corresponding search if needed.
+ */
+const BiosampleCharacterizationTableFooter = ({ items, total, url }) => (
+    <div>
+        <span>Displaying {items.length} of {total} </span>
+        {items.length < total ? <a className="btn btn-info btn-xs pull-right" href={url}>View all</a> : null}
+    </div>
+);
```

Comment:
React 16 has a new feature to replaced these extra `<div>`s — only needed for JSX parsing — with `<React.Fragment>`...`</React.Fragment>`. These serve the same purpose as the `<div>`s but don’t generate any actual HTML elements.